src/ip/Address.h

```diff
@@ -298,6 +298,10 @@ class Address
      */
     bool GetHostByName(const char *s);
 
+    /// \returns an object containing the specific IP case NO_ADDR (format-neutral).
+    /// see isNoAddr() for more detail.
```

Comment:
I know you are copying this phrase, but I would polish it:

```suggestion
    /// \see isNoAddr() for more details
```